app/dashboard/templates/bounty/new.html

```diff
@@ -170,5 +171,7 @@ <h5 class="font-subheader pb-2">{% trans "Gas Settings" %}</h4>
 <script src="{% static "v2/js/shared.js" %}"></script>
 <script src="{% static "v2/js/pages/shared_bounty_mutation_estimate_gas.js" %}"></script>
 <script src="{% static "v2/js/pages/new_bounty.js" %}"></script>
+<script src="{% static "v2/js/jquery.select2.min.js" %}"></script>
```

Comment:
This template already `includes` shared/footer_scripts.html which already has `v2/js/jquery.select2.min.js` in it.  This can be removed.